phone_calls/python/test.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
```

Comment:
@vmax given that `test.py` here is called via a `py_test`, do we still need this line?

Reply:
@sorsaffari we actually don't; though I don't think it's bad practice to still specify it